releases/net471/KnownIssues/517815-BCL Applications making heavy use of System.Diagnostics.StackTrace might run more slowly on .NET 4.7.1.md

```diff
@@ -0,0 +1,19 @@
+# BCL Applications making heavy use of System.Diagnostics.StackTrace or Exception.StackTrace might run more slowly on .NET 4.7.1
+
+## Symptoms
+Application that had acceptable performance running on .NET 4.7 executes more slowly when running on .NET 4.7.1
+
+## Cause
+.NET 4.7.1 added support for detecting and parsing the Portable PDB format to show file and line number information in stack traces. As part of this change each function in a 
+stack trace needed to have its defining module checked to determine if that module used the Portable PDB format. Due to some differences in the internal caching policy the 
```

Comment:
This reads much better if the tense here is present. So:
needed to have its defining module --> has its defining module
used --> uses
comma after policy